docs/api/oauth.md

```diff
@@ -0,0 +1,445 @@
+---
+sidebar_label: Log in with Audius
```

Comment:
In these docs I refer to "readonly OAuth" as Log in with Audius - I didn't call it OAuth because it isn't really OAuth (since no permissions are granted). Is there a better name though?

Reply:
This is a tough one bc `Login w Audius` does kinda sound like it will manage a login session for them. But I think it's fine for now